### DISCUSSION TITLE
Just wanted to say thanks to the devs for such an Epic app

### DISCUSSION THREAD

**Discussion by stalta** [WindowTop/WindowTop-App#387] [General]
Just wanted to say thanks to the devs, this app is incredible, it was just what i was looking for, i could tell alot of work was put into it :)

**Comment by gileli121** [comment]
Thank you very much for this feedback!
One of the main reasons I keep putting effort into this is the gratitude from users who appreciate the product :)

I would appreciate it if you spread the word and recommend this app to anyone who might need it.
Enjoy WindowTop and have a great day!
